app/src/main/java/com/example/concordia_campus_guide/LocationFragment/LocationFragment.java

```diff
@@ -112,6 +140,12 @@ public void onMapReady(GoogleMap googleMap) {
      * The purpose of this method is to figure the style of the map to display
      * @param googleMap is the map that is used in the application
      */
+    private void initFloorPlans() {
```

Comment:
Perhaps we could store these in a ground overlay array, so that our code can grow without needing to rewrite the code?